aspnetcore/migration/22-to-30.md

```diff
@@ -643,4 +643,4 @@ If you're using features of `Newtonsoft.Json` that aren't supported in `System.T
 
 ## Opt in to runtime compilation
 
-In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see https://docs.microsoft.com/aspnet/core/mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation.
+In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see <xref:mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation>.
```

Comment:
The referenced doc is small enough that it's title should serve well here. I didn't want to say "runtime compilation" 3 times in two lines.